### PR TITLE
add pointcloud object

### PR DESCRIPTION
**Pull request by Licini**
#114

### CODE REVIEW

File: docs/examples/object/pointcloud.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from random import random
+
+from compas.colors import Color
+from compas.geometry import Pointcloud
+from compas_viewer import Viewer
+
+viewer = Viewer()
+pointcloud = Pointcloud.from_bounds(10, 10, 10, 1000)
+viewer.scene.add(pointcloud, pointcolor=Color(1.0,0.0,0.0), pointssize=10)
```
Comment (tomvanmele) [inline docs/examples/object/pointcloud.py:9]: possible to set different colors/sizes per point?

Comment (Licini) [reply docs/examples/object/pointcloud.py:9]: pointcolor should be relatively easy... will double check. But size might involve some shader update

Comment (Licini) [reply docs/examples/object/pointcloud.py:9]: the pointcolor input should be able to take dict of colors as we initially designed, but now the whole config thing is also interacting with it in someway and stoping me to do that.....

Comment (tomvanmele) [reply docs/examples/object/pointcloud.py:9]: okay not important now but should be added in the future

**Review by tomvanmele — APPROVED** [review]

**Pull request merged**